SuperNewRoles/Resources/Translate.csv

```diff
@@ -1748,56 +1748,56 @@ CursedWeaponsTaskScoreText,Destroyed : ({0}/{1}),破壊済み : ({0}/{1}),破坏
 
 # Other
 WelcomeMessage1,Welcome to SuperNewRoles!,SuperNewRolesへようこそ！,欢迎游玩Super New Roles（超新职业）！,歡迎遊玩Super New Roles（超新執業）！
-WelcomeMessage2,This is the room where we installed a mod called SuperNewRoles.,この部屋はSuperNewRolesというModを導入しています。
-WelcomeMessage3,This mod is not affiliated with Among Us or Innersloth LLC，and the content contained therein is not endorsed or otherwise sponsored by Innersloth LLC.,当ModはAmong UsやInnersloth LLCとは無関係であり、含まれるコンテンツはInnersloth LLCによって承認されたり、直接提供を受けているものではありません。
-WelcomeMessage4,Please contact the official SuperNewRoles Discord server for any impact，damage or bugs that may occur in this room，not Innersloth.,この部屋で発生した，影響や損害及びバグについてはInnerslothに問い合わせず，SuperNewRoles公式 Discordサーバに問い合わせてください。
-WelcomeMessage5,You can play a variety of modes and different hard on any model with SuperNewRoles!,SNRを使用することで、様々なモードや様々な役職をどの機種でも遊べます！,您可以在超新职业体验各种不同的模式和职业！,您可以在超新職業體驗各種不同的模式和職業！
+WelcomeMessage2,This is the room where we installed a mod called SuperNewRoles.,この部屋はSuperNewRolesというModを導入しています。,,已安裝了超新職業模組。
+WelcomeMessage3,This mod is not affiliated with Among Us or Innersloth LLC，and the content contained therein is not endorsed or otherwise sponsored by Innersloth LLC.,当ModはAmong UsやInnersloth LLCとは無関係であり、含まれるコンテンツはInnersloth LLCによって承認されたり、直接提供を受けているものではありません。,,此模組與Among Us或Innersloth LLC無關，其中包含的內容未經Innersloth LLC批准或贊助。
+WelcomeMessage4,Please contact the official SuperNewRoles Discord server for any impact，damage or bugs that may occur in this room，not Innersloth.,この部屋で発生した，影響や損害及びバグについてはInnerslothに問い合わせず，SuperNewRoles公式 Discordサーバに問い合わせてください。,,如果遊玩過程中發生任何問題或錯誤，請聯繫 SNR 超新職業官方Discord服務器，而非Innersloth。
+WelcomeMessage5,You can play a variety of modes and different hard on any model with SuperNewRoles!,SNRを使用することで、様々なモードや様々な役職をどの機種でも遊べます！,您可以在超新职业体验各种不同的模式和职业！,您可以在超新職業體驗各種不同的模式和職業！,,您可以在超新職業模組體驗各種不同的模式和職業！
```

Comment:
Since we have only changed the translation key here, all the original languages are still used.
We would appreciate it if you could change the description to only those that are to be used.

Reply:
I am sorry that I did not notice that the original texts are there. I have removed it.
Thanks for reminding me.